src/mumble/MainWindow.cpp

```diff
@@ -756,6 +838,9 @@ void MainWindow::changeEvent(QEvent *e) {
 		QTimer::singleShot(0, this, SLOT(hide()));
 	}
 #endif
+	if (e->type() == QEvent::ThemeChange) {
```

Comment:
We have to assume that Qt sends us many of these events for no particular reason (Qt sucks sometimes). So your code should - at some point preferably in ``Themes.cpp`` - check if we are already on the theme that Qt wants to change to and abort early. This will prevent reloading the same theme over and over again which I imagine could be quite glitchy and/or annoying.